styles/base/_colors.scss

```diff
@@ -9,6 +9,9 @@ $blue: #36a;
 $purple: #636;
 $rose: #957;
 $red: #a22;
+$orange: #a36036;
+$yellow: #a1a336;
```

Comment:
@willuhm-js The orange looks great, but the yellow is a bit too strong. Could we try a bit more gold-ish yellow like `#a83`?

<img width="1087" alt="Screen Shot 2022-02-05 at 12 38 18 PM" src="https://user-images.githubusercontent.com/872474/152658213-eb9c8aeb-7ed6-4b52-a56d-8720c86a75fa.png">